src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorIndentationFactsService.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Would have loved to change this API to:
`bool TryGetDesiredIndentation(RazorSyntaxTree syntaxTree, int lineIndex, int indentSize, int tabSize)` but [this](https://github.com/aspnet/Razor/issues/1876) is preventing me from doing that.

Reply:
Can you add a note to that bug explaining what to clean up once it's fixed?

Reply:
Will do